### PR TITLE
Modify TiKV in 5 Minutes

### PR DESCRIPTION
**Pull request by TomShawn**
Signed-off-by: TomShawn <41534398+TomShawn@users.noreply.github.com>

### What is changed?
Modify TiKV in 5 Minutes
Related PR: https://github.com/tikv/website/pull/224

### CODE REVIEW

File: content/docs/5.1/concepts/tikv-in-5-minutes.md [hunk 1]
```diff
@@ -30,139 +32,143 @@ The following steps are only for test environments. For production environments
...
+    2. Confirm whether TiUP is installed:
 
-    ```bash
-    tiup
-    ```
+        ```bash
+        tiup
+        ```
+
+    <!-- Explain the results -->
```
Comment (TomShawn) [inline content/docs/5.1/concepts/tikv-in-5-minutes.md:47]: @marsishandsome Should we explain what the expected result is?

File: content/docs/5.1/concepts/tikv-in-5-minutes.md [hunk 2]
```diff
@@ -30,139 +32,143 @@ The following steps are only for test environments. For production environments
...
+2. Open the `TiKV-Summary` page on the Grafana Dashboard and find the monitoring metrics of your TiKV cluster.
 
-1. Download jars
+## Write data to and read data from the TiKV cluster
 
-    ```bash
-    curl -o tikv-client-java.jar https://download.pingcap.org/tikv-client-java-3.2.0-SNAPSHOT.jar
-    curl -o slf4j-api.jar https://repo1.maven.org/maven2/org/slf4j/slf4j-api/1.7.16/slf4j-api-1.7.16.jar
-    ```
+To write to and read from the TiKV cluster, you can use either Java or Python script. In the following two examples, Java and Python are respectively used to write "Hello World!" to TiKV.
```
Comment (TomShawn) [inline content/docs/5.1/concepts/tikv-in-5-minutes.md:87]: @marsishandsome Are Python and Java the only two ways to write and read?

File: content/docs/5.1/concepts/tikv-in-5-minutes.md [hunk 3]
```diff
@@ -30,139 +32,143 @@ The following steps are only for test environments. For production environments
...
+        value: "Hello"
+        , key: "k2"
+        value: ","
+        , key: "k3"
+        value: "World"
+        , key: "k4"
+        value: "!"
+        , key: "k5"
+        value: "Raw KV"
+        ]
```
Comment (TomShawn) [inline content/docs/5.1/concepts/tikv-in-5-minutes.md:166]: Is this part the output or what we should write in the command?

Comment (marsishandsome) [reply content/docs/5.1/concepts/tikv-in-5-minutes.md:166]: yes

File: content/docs/5.1/concepts/tikv-in-5-minutes.md (hunk 2 above)
Comment (marsishandsome) [reply content/docs/5.1/concepts/tikv-in-5-minutes.md:87]: There will be another language APIs, e.g. go c++. But it's not ready now.

File: content/docs/5.1/concepts/tikv-in-5-minutes.md (hunk 1 above)
Comment (marsishandsome) [reply content/docs/5.1/concepts/tikv-in-5-minutes.md:47]: Please delete this line.

Comment (TomShawn) [reply content/docs/5.1/concepts/tikv-in-5-minutes.md:47]:
```suggestion
```

File: content/docs/5.1/concepts/tikv-in-5-minutes.md (hunk 3 above)
Comment (TomShawn) [reply content/docs/5.1/concepts/tikv-in-5-minutes.md:165]: Is this the output?

OR

Is this what we should write in the command?

File: content/docs/5.1/concepts/tikv-in-5-minutes.md [hunk 4]
```diff
@@ -7,20 +7,22 @@ menu:
         weight: 1
 ---
 
-This document provides a fast-lane tutorial to get started with TiKV, which includes:
-- setting up a local TiKV cluster with the default options,
-- monitoring the TiKV cluster,
-- writing data to and reading data from the TiKV cluster.
+This tutorial provides you a quick way to get started with TiKV. You are guided to perform the following operations:
```
Comment (TomShawn) [inline content/docs/5.1/concepts/tikv-in-5-minutes.md:10]:
```suggestion
This tutorial provides you a quick way to get started with TiKV. You are guided to perform the following operations: 
```

File: content/docs/5.1/concepts/tikv-in-5-minutes.md (hunk 3 above)
Comment (marsishandsome) [reply content/docs/5.1/concepts/tikv-in-5-minutes.md:164]: it's output

File: content/docs/5.1/concepts/tikv-in-5-minutes.md [hunk 5]
```diff
@@ -30,139 +32,141 @@ The following steps are only for test environments. For production environments
...
-
-    Show TiUP version:
+4. Use TiUP Playground to start a local TiKV cluster. Before you do that, check your TiUP version using the following command:
 
     ```bash
     tiup -v
     ```
 
-    version >= 1.5.0:
+    - If your TiUP version is v1.5.0 or later, execute the following command to start a local TiKV cluster:
```
Comment (marsishandsome) [inline content/docs/5.1/concepts/tikv-in-5-minutes.md:59]:
```suggestion
    - If your TiUP version is v1.5.2 or later, execute the following command to start a local TiKV cluster:
```

File: content/docs/5.1/concepts/tikv-in-5-minutes.md [hunk 6]
```diff
@@ -30,139 +32,141 @@ The following steps are only for test environments. For production environments
...
 
-    ```bash
-    tiup playground --mode tikv-slim
-    ```
+        ```bash
+        tiup playground --mode tikv-slim
+        ```
 
-    version < 1.5.0:
+    - If your TiUP version is earlier than v1.5.0, execute the following command to start a local TiKV cluster:
```
Comment (marsishandsome) [inline content/docs/5.1/concepts/tikv-in-5-minutes.md:65]:
```suggestion
    - If your TiUP version is earlier than v1.5.2, execute the following command to start a local TiKV cluster:
```

File: content/docs/5.1/concepts/tikv-in-5-minutes.md (hunk 2 above)
Comment (marsishandsome) [reply content/docs/5.1/concepts/tikv-in-5-minutes.md:85]:
```suggestion
To write to and read from the TiKV cluster, you can use either Java, Go, Rust, C or Python script. In the following two examples, Java and Python are respectively used to write "Hello World!" to TiKV.
```

File: content/docs/5.1/concepts/tikv-in-5-minutes.md (hunk 3 above)
Comment (TomShawn) [reply content/docs/5.1/concepts/tikv-in-5-minutes.md:168]: It would be better to separate the output and the command.

Comment (marsishandsome) [reply content/docs/5.1/concepts/tikv-in-5-minutes.md:168]: agree

**Comment by TomShawn** [comment]
@marsishandsome @qiancai PTAL again, thanks!

**Review by marsishandsome — APPROVED** [review]
LGTM

File: content/docs/5.1/concepts/tikv-in-5-minutes.md [hunk 7]
```diff
@@ -7,20 +7,22 @@ menu:
...
-This document provides a fast-lane tutorial to get started with TiKV, which includes:
-- setting up a local TiKV cluster with the default options,
-- monitoring the TiKV cluster,
-- writing data to and reading data from the TiKV cluster.
+This tutorial provides you a quick way to get started with TiKV. You are guided to perform the following operations:
+
+- [Set up a local TiKV cluster with the default options](#set-up-a-local-tikv-cluster-with-the-default-options)
+- [Monitor the TiKV cluster](#monitor-the-cluster)
+- [Write data to and read data from the TiKV cluster](#write-data-to-and-read-data-from-the-tikv-cluster)
+- [Stop and deleted a TiKV cluster](#stop-and-delete-the-tikv-cluster)
```
Comment (Liuxiaozhen12) [inline content/docs/5.1/concepts/tikv-in-5-minutes.md:15]:
```suggestion
- [Stop and delete a TiKV cluster](#stop-and-delete-the-tikv-cluster)
```
Found a small typo when learning from your review~

File: content/docs/5.1/concepts/tikv-in-5-minutes.md [hunk 8]
```diff
@@ -30,139 +32,145 @@ The following steps are only for test environments. For production environments
...
+1. Open your browser, access [http://127.0.0.1:3000](http://127.0.0.1:3000), and login to the Grafana Dashboard. By default, the username is `admin` and the password is `admin`.
 
-## Step 3: Hello, World!
+2. Open the `TiKV-Summary` page on the Grafana Dashboard and find the monitoring metrics of your TiKV cluster.
 
-### Java
+## Write data to and read data from the TiKV cluster
 
-1. Download jars
+To write to and read from the TiKV cluster, you can use Java, Go, Rust, C or Python script. In the following two examples, Java and Python are respectively used to write "Hello World!" to TiKV.
```
Comment (qiancai) [inline content/docs/5.1/concepts/tikv-in-5-minutes.md:85]:
```suggestion
To write to and read from the TiKV cluster, you can use Java, Go, Rust, C, or Python script. 

The following two examples use Java and Python respectively to show how to write "Hello World!" to TiKV.
```

File: content/docs/5.1/concepts/tikv-in-5-minutes.md [hunk 9]
```diff
@@ -30,139 +32,145 @@ The following steps are only for test environments. For production environments
...
+        // scan
+        list = client.scan(ByteString.copyFromUtf8("k1"), ByteString.copyFromUtf8("k6"), 10);
+        System.out.println(list);
+
+        // close
+        client.close();
+        session.close();
+        ```
+
+    2. Run the script above to write "Hello World!" to TiKV:
```
Comment (qiancai) [inline content/docs/5.1/concepts/tikv-in-5-minutes.md:143]:
```suggestion
    2. Run the script in the `test_raw.java` file to write "Hello World!" to TiKV:
```

File: content/docs/5.1/concepts/tikv-in-5-minutes.md [hunk 10]
```diff
@@ -30,139 +32,145 @@ The following steps are only for test environments. For production environments
...
+
+        ```bash
+        jshell --class-path tikv-client-java.jar:slf4j-api.jar --startup test_raw.java
+        ```
+
+        The following result is output:
```
Comment (qiancai) [inline content/docs/5.1/concepts/tikv-in-5-minutes.md:149]:
```suggestion
        The output is as follows:
```

File: content/docs/5.1/concepts/tikv-in-5-minutes.md [hunk 11]
```diff
@@ -172,88 +180,98 @@ Open `TiKV-Summary` page on Grafana to find the metrics of your TiKV cluster.
...
+            print(client.batch_get([b"k1", b"k3"]))
 
-    Run the test script
+            # scan
+            print(client.scan(b"k1", end=b"k5", limit=10, include_start=True, include_end=True))
+            ```
 
-    ```bash
-    python3 test_raw.py
+        2. Run the test script above to write "Hello World!" to TiKV:
```
Comment (qiancai) [inline content/docs/5.1/concepts/tikv-in-5-minutes.md:211]:
```suggestion
        2. Run the `test_raw.py` script to write "Hello World!" to TiKV:
```

File: content/docs/5.1/concepts/tikv-in-5-minutes.md [hunk 12]
```diff
@@ -172,88 +180,98 @@ Open `TiKV-Summary` page on Grafana to find the metrics of your TiKV cluster.
...
-    b'Hello'
-    [(b'k3', b'World'), (b'k1', b'Hello')]
-    [(b'k1', b'Hello'), (b'k2', b','), (b'k3', b'World'), (b'k4', b'!'), (b'k5', b'Raw KV')]
-    ```
+            ```bash
+            python3 test_raw.py
+            ```
 
-3. Try `TXN KV` API
+            The following result is output.
```
Comment (qiancai) [inline content/docs/5.1/concepts/tikv-in-5-minutes.md:217]:
```suggestion
            The output is as follows:
```

File: content/docs/5.1/concepts/tikv-in-5-minutes.md [hunk 13]
```diff
@@ -172,88 +180,98 @@ Open `TiKV-Summary` page on Grafana to find the metrics of your TiKV cluster.
...
-    python3 test_txn.py
+            # scan
+            print(snapshot.scan(b"k1", end=b"k5", limit=10, include_start=True, include_end=True))
+            ```
 
-    b'Hello'
-    [(b'k3', b'World'), (b'k1', b'Hello')]
-    [(b'k1', b'Hello'), (b'k2', b','), (b'k3', b'World'), (b'k4', b'!'), (b'k5', b'TXN KV')]
-    ```
+        2. Run the test script to write "Hello World!" to TiKV:
```
Comment (qiancai) [inline content/docs/5.1/concepts/tikv-in-5-minutes.md:255]:
```suggestion
        2. Run the `test_txn.py` script to write "Hello World!" to TiKV:
```

File: content/docs/5.1/concepts/tikv-in-5-minutes.md [hunk 14]
```diff
@@ -172,88 +180,98 @@ Open `TiKV-Summary` page on Grafana to find the metrics of your TiKV cluster.
...
+            ```bash
+            b'Hello'
+            [(b'k3', b'World'), (b'k1', b'Hello')]
+            [(b'k1', b'Hello'), (b'k2', b','), (b'k3', b'World'), (b'k4', b'!'), (b'k5', b'TXN KV')]
+            ```
 
-2. You can destroy the cluster by:
+## Stop and delete the TiKV cluster
+
+1. To stop the TiKV cluster, back to the terminal session in which you have started the TiKV cluster. Press <kbd>Ctrl</kbd> + <kbd>C</kbd> and wait for the cluster to stop.
```
Comment (qiancai) [inline content/docs/5.1/concepts/tikv-in-5-minutes.md:271]:
```suggestion
If you do not need the local TiKV cluster anymore, you can stop and delete it.

1. To stop the TiKV cluster, get back to the terminal session in which you have started the TiKV cluster. Press <kbd>Ctrl</kbd> + <kbd>C</kbd> and wait for the cluster to stop.
```

File: content/docs/5.1/concepts/tikv-in-5-minutes.md (hunk 4 above)
Comment (qiancai) [inline content/docs/5.1/concepts/tikv-in-5-minutes.md:10]:
```suggestion
This tutorial provides you a quick way to get started with TiKV, including the following operations:
```

**Review by qiancai — COMMENTED** [review]
@TomShawn Good writing. I just have some minor comments, PTAL.

File: content/docs/5.1/concepts/tikv-in-5-minutes.md [hunk 15]
```diff
@@ -172,88 +180,98 @@ Open `TiKV-Summary` page on Grafana to find the metrics of your TiKV cluster.
...
-    [(b'k1', b'Hello'), (b'k2', b','), (b'k3', b'World'), (b'k4', b'!'), (b'k5', b'TXN KV')]
-    ```
+        2. Run the test script to write "Hello World!" to TiKV:
+
+            ```bash
+            python3 test_txn.py
+            ```
 
-## Step 4: Stop and delete the cluster.
+            The following result is output:
```
Comment (qiancai) [inline content/docs/5.1/concepts/tikv-in-5-minutes.md:261]:
```suggestion
            The output is as follows:
```

**Comment by TomShawn** [comment]
@qiancai All comments are accepted. PTAL again, thanks!

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**